### DISCUSSION TITLE
Excluding low-frame-count utterances

### DISCUSSION THREAD

**Discussion by feddybear** [espnet/espnet#3724] [Q&A] [answered]
What's the best practice to exclude entry utterances that have frames that are too low for subsampling?

**Comment by sw005320** [comment, marked as answer]
We have a stage to remove such files https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/asr.sh#L553
You can manually specify `min_wav_duration` to exclude too short utterances in the data preparation stage.
